docs/modules/ROOT/pages/operations.adoc

```diff
@@ -272,7 +272,7 @@ Possible values: `auto`, `normal`, `outline`, and `dark`.
 
 | `playImplicitInitCommands`
 | `true`
-| TODO: Commands that are run when connected to the graph.
+| Allow you to run some commands automatically on startup, for example, `:play start` and `:server connect`.
```

Comment:
Perhaps clearer as `If set to true browser will automatically run some commands on startup, for example :play start and :server connect`

Reply:
Does it run the command that is set in `initCmd`?

Reply:
If so, I can rephrase it to "If `true`, Browser automatically runs the command set in `initCmd` on startup.

Reply:
yes